web/docs/operators/package.md

```diff
@@ -26,24 +26,30 @@ and user-facing documentation of the package format is still in progress.
 ```
 package add
 package remove <package_id>
+packages [--format=<format>]
```

Comment:
I think this should have a separate documentation page for consistency with the other operators now.